awx_collection/plugins/modules/controller_user.py

```diff
@@ -71,43 +71,43 @@
 
 
 EXAMPLES = '''
-- name: Add tower user
-  tower_user:
+- name: Add controller user
+  controller_user:
     username: jdoe
     password: foobarbaz
     email: jdoe@example.org
     first_name: John
     last_name: Doe
     state: present
-    tower_config_file: "~/tower_cli.cfg"
+    controller_config_file: "~/tower_cli.cfg"
 
-- name: Add tower user as a system administrator
-  tower_user:
+- name: Add controller user as a system administrator
+  controller_user:
     username: jdoe
     password: foobarbaz
     email: jdoe@example.org
     superuser: yes
     state: present
-    tower_config_file: "~/tower_cli.cfg"
+    controller_config_file: "~/tower_cli.cfg"
```

Comment:
should we change to `controller_cli.cfg`?

Reply:
the modules in this collection predate AWX itself. When they were created, Ansible Tower existed and nothing else.

Several things here resisted the name change from Tower->AWX, due to risks of breaking people's content. The config file is possibly the most painful example of that. It uses the location and formats from the `tower-cli` project, which was replaced with the `awx-cli`. The CLI itself has no notion of a config file itself, but the modules (which became the collection) still respects the naming and format from 2014

Reply:
see /etc/tower/SECRET_KEY. This is probably not worth changing.